.github/workflows/CI-production-testing.yml

```diff
@@ -167,7 +167,11 @@ jobs:
         coverage run --source=./ -m pytest -s tests/integration_tests/test_portscans.py -p no:warnings -vv
         coverage report --include="modules/network_discovery/*"
         coverage html --include="modules/network_discovery/*" -d coverage_reports/network_discovery
-
+    
+    - name: set evidence test
+      run: |
+        coverage run --source=./ -m pytest tests/test_set_evidence.py -p no:warnings -vv
+         
```

Comment:
this command is missing
coverage report --include=""
 coverage html --include="" -d 
 should be exactly like the 3 lines above

Reply:
okay